master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +427,24 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
 outbound traffic *from* port 999.  (Whereas for a connection tracked policy, it
 is usually enough to specify the ingress rule only, and then connection
 tracking will automatically allow the return path.)
+
+## Pre-DNAT policy
+
+Policy for host endpoints can be marked as 'preDNAT'.  This means that rules in
```

Comment:
Very well written section. One suggestion, consider expanding DNAT acronym once (if it's not already defined anywhere else) - I know not everyone is familiar with the term.